src/frontend/e2e/utils/sample.ts

```diff
@@ -127,19 +137,19 @@ export interface SampleUploadData {
   };
 }
 
-export interface GetSampleResponseData {
+export interface SampleResponseData {
   id: number;
   collection_date: string;
   collection_location: {
-    id: number;
+    id?: number;
     region: string;
     country: string;
     division: string;
     location: string;
   };
   czb_failed_genome_recovery: boolean;
   gisaid: {
-    gisaid_id: string;
+    gisaid_id: any;
```

Comment:
why is this any type?

Reply:
Currently this comes as null from backend (https://app.shortcut.com/genepi/story/216575/potential-typo-in-sample-response-from-backend) and I don't know to expect.
I have changed it to string | null for now